pkg/multiclustermanager/capabilities/gke_version_endpoint.go

```diff
@@ -1,49 +0,0 @@
-package capabilities
-
-import (
-	"fmt"
-	"net/http"
-)
-
-const (
-	defaultCredentialEnv = "GOOGLE_APPLICATION_CREDENTIALS"
-)
-
-// NewGKEVersionsHandler creates a new GKEVersionsHandler
-func NewGKEVersionsHandler() *GKEVersionHandler {
-	return &GKEVersionHandler{}
-}
-
-// GKEVersionHandler for listing available Kubernetes versions in GKE
-type GKEVersionHandler struct {
-}
-
-type errorResponse struct {
-	Error string `json:"error"`
-}
-
-func (g *GKEVersionHandler) ServeHTTP(writer http.ResponseWriter, req *http.Request) {
-	body := preCheck(writer, req)
-	if body == nil {
-		return
-	}
-
-	zone := body.Zone
-	if zone == "" {
-		writer.WriteHeader(http.StatusBadRequest)
-		handleErr(writer, fmt.Errorf("invalid zone"))
```

Comment:
This validation is missing now, it should be added to `listZones`

Reply:
I have added it to proper functions

Reply:
:+1: `listZones` was wrong, you added them to the right place